pkgs/development/libraries/gdal/default.nix

```diff
@@ -20,22 +20,22 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "source/gdal";
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
 
   buildInputs = [ unzip libjpeg libtiff libpng proj openssl sqlite
     libspatialite libgeotiff poppler hdf4 qhull giflib expat libxml2 ]
-  ++ (with pythonPackages; [ python numpy wrapPython ])
-  ++ stdenv.lib.optional stdenv.isDarwin libiconv
-  ++ stdenv.lib.optionals netcdfSupport [ netcdf hdf5 curl ];
+    ++ (with pythonPackages; [ python numpy wrapPython ])
```

Comment:
Is Python 3 `python3Packages` not supported ?

Reply:
I can test. Didn't tinker around with this to be honest. I did this change, because I wasn't able to use ogr2ogr with PG plugin.

Reply:
I have made this change, the build passes fine and everything is working for me correctly.

Reply:
Turns out `python-packages.nix` override this attribute so we would need to change it in `all-packages.nix` instead:

```
  gdal = callPackage ../development/libraries/gdal {
    pythonPackages = python3Packages;
  };
```

Reply:
I have included this adjustment.